docs/library/uarray.rst

```diff
@@ -6,6 +6,11 @@
 
 |see_cpython_module| :mod:`python:array`.
 
+This module, ``uarray``, is a subset of ``python:array`` and only the methods documented below are available, 
+with the exception of ``decode``.
```

Comment:
This sentence duplicates most of what is in the `see_cpython_module` replacement-macro just above.